diff --git a/src/App/Branding.cpp b/src/App/Branding.cpp
index 6f59302..1fd5ef8 100644
--- a/src/App/Branding.cpp
+++ b/src/App/Branding.cpp
@@ -45,16 +45,22 @@ Branding::Branding()
     filter.push_back("BuildVersionMinor");
     filter.push_back("BuildRevision");
     filter.push_back("BuildRevisionDate");
+    filter.push_back("BuildVersionSuffix");
+    filter.push_back("BuildRepositoryURL");
 
+    filter.push_back("AboutImage");
     filter.push_back("SplashScreen");
     filter.push_back("SplashAlignment");
     filter.push_back("SplashTextColor");
     filter.push_back("SplashInfoColor");
+    filter.push_back("SplashInfoFont");
+    filter.push_back("SplashInfoPosition");
 
     filter.push_back("StartWorkbench");
 
     filter.push_back("ExeName");
     filter.push_back("ExeVendor");
+    filter.push_back("AppDataSkipVendor");
     filter.push_back("NavigationStyle");
     filter.push_back("UserParameterTemplate");
 }
